GoMeet/src/test/java/com/google/sps/ServletTestUtil.java

```diff
@@ -0,0 +1,48 @@
+package test.java.com.google.sps;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+import com.google.gson.JsonParser;
+import com.google.gson.JsonObject;
+import com.google.sps.data.ServletUtil;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.HashMap;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Class for servlet unit testing helper functions 
+ */
+public class ServletTestUtil {
+  
+  /**
+   * Helper function to test the correct error response was sent in the case of 
+   * a bad request
+   */
+  public static void badRequest(int status, String errorMessage, 
```

Comment:
I would call this function "expectBadRequest" or similar because it has some assertions that are meant to pass inside it.